libafl/src/corpus/mod.rs

```diff
@@ -36,8 +36,8 @@ where
     /// Add an entry to the corpus and return its index
     fn add(&mut self, testcase: Testcase<I>) -> Result<usize, Error>;
 
-    /// Replaces the testcase at the given idx
-    fn replace(&mut self, idx: usize, testcase: Testcase<I>) -> Result<(), Error>;
+    /// Replaces the testcase at the given idx, returning the existing.
+    fn replace(&mut self, idx: usize, testcase: Testcase<I>) -> Result<Testcase<I>, Error>;
```

Comment:
Potentially breaking: replace now returns the test case which previously occupied that index.

Reply:
Don't think anyone used it till now, should be good